### PR TITLE
chore: bump samples 1.5.0 m4

### PR DESCRIPTION
**Pull request by johanandren**
Also, include spotless formatting in default build and reformat java samples

### CODE REVIEW

File: samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala [hunk 1]
```diff
@@ -26,17 +27,14 @@ object DeliveryEvents {
 
     val eventsBySlicesQuery =
       GrpcReadJournal(
+        GrpcQuerySettings(system).withInitialConsumerFilter(
+          // location id already is in the format of a topic filter expression
+          Vector(
+            ConsumerFilter.ExcludeRegexEntityIds(Set(".*")),
+            ConsumerFilter.IncludeTopics(Set(settings.locationId)))),
+        GrpcClientSettings.fromConfig("akka.projection.grpc.consumer.client"),
```
Comment (johanandren) [inline samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala:35]: Still somewhat inconvenient, we should add another GrpcReadJournal.apply/create overload where you don't have to specify GrpcClientSettings.

Comment (johanandren) [reply samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala]: Actually even worse than I first thought, I'll fix.

Comment (johanandren) [reply samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala]: Fix in https://github.com/akka/akka-projection/pull/998

File: samples/grpc/local-drone-control-java/src/main/java/local/drones/DeliveryEvents.java [hunk 2]
```diff
@@ -35,20 +37,15 @@ public static Behavior<ProjectionBehavior.Command> projectionBehavior(
         var eventsBySlicesQuery =
             GrpcReadJournal.create(
                 system,
+                GrpcQuerySettings.create(system).withInitialConsumerFilter(
+                    // location id already is in the format of a topic filter expression
+                    Arrays.asList(
+                            new ConsumerFilter.ExcludeRegexEntityIds(Collections.singleton(".*")),
```
Comment (patriknw) [inline samples/grpc/local-drone-control-java/src/main/java/local/drones/DeliveryEvents.java]: We could use the new ExcludeAll (in all places where we use `.*`)

**Review by patriknw — APPROVED** [review]
LGTM

File: samples/grpc/local-drone-control-java/src/main/java/local/drones/DeliveryEvents.java (hunk 2 above)
Comment (johanandren) [reply samples/grpc/local-drone-control-java/src/main/java/local/drones/DeliveryEvents.java]: Ah, I had forgotten that one, I'll replace this. 👍

**Comment by patriknw** [comment]
merging this, let's change the ExcludeAll as follow up

**Pull request merged**